backend/seeding/seedEntities.ts

```diff
@@ -0,0 +1,47 @@
+import { faker } from "@faker-js/faker";
+import { MongoClient } from "mongodb";
+import { Entity } from "../models/entity.model";
+
+function createRandomEntity(): Entity {
+  return {
+    id: faker.string.uuid(),
+    stringField: faker.word.words({ count: { min: 1, max: 10 } }),
+    intField: faker.number.int({ min: 2015, max: 2024 }),
+    enumField: faker.helpers.arrayElement(["A", "B", "C", "D"]),
+    stringArrayField: faker.helpers.arrayElements([
+      "One",
+      "Two",
+      "Three",
+      "Four",
+      "Five",
+    ]),
+    boolField: faker.datatype.boolean(),
+    fileName: faker.word.noun(),
+  } as Entity;
+}
+
+async function seedEntities(): Promise<void> {
+  const uri =
+    "mongodb+srv://cwc:jEkj8yE3mOWeviIO@child-witness-centre.6c4i8ne.mongodb.net/";
```

Comment:
I think this should be set using the environment variables, but it keeps showing up as undefined after using dotenv config so I left it like this. Would appreciate any suggestions!